src/api/dispatch/dispatch.ts

```
@@ -1,11 +1,22 @@
 import { createApiEndpoint } from '@/api/common/client';
+import { type NewCallFormResult } from '@/models/v4/dispatch/newCallFormResult';
 import { type GetSetUnitStateResult } from '@/models/v4/dispatch/getSetUnitStateResult';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)